cogs/FamRankings.py

```diff
@@ -1,33 +1,33 @@
+import json
+import random
+from datetime import datetime as dt
 import discord
 from discord.ext import commands
-from structs.rankings import fam_by_rank, rank_title
-from structs.responses import memes
 import progressbar as pb
-import random
-from datetime import datetime as dt
-
+from structs.responses import memes
+from structs.rankings import fam_by_rank, rank_title, famDict
 
-import json
 
-famDict = {
-    "isfam": [
-        "WikiWikiWasp",
-        "Wirt.zirp",
-        "bossanova",
-        "Snail",
-        "The Mongoose",
-        "shaggyzero",
-        "ToeUp"
-    ],
-    "jsquad": [
-        "WikiWikiWasp",
-        "Wirt.zirp",
-        "shaggyzero",
-    ]
-}
+# famDict = {
```

Comment:
Don't just comment old code, delete old code!

Reply:
this was laziness lol it can be deleted, iirc

Reply:
deleted